configs/development.ini

```diff
@@ -1,9 +1,9 @@
 [bot]
 description = thonkbot development mode
 environment = development
-secrets = local/secrets.ini
+secrets = secrets/secrets.ini
 command_prefixes = ~
 log_level = DEBUG
 
 [twitter]
-config = local/twitter.json
+config = secrets/twitter.json
```

Comment:
I think I'd prefer this folder name to be reverted back to `local` - it's just meant to be for any extra config or data files used for development